example-cluster/infrastructure/common.yml

```diff
@@ -41,3 +41,14 @@ asg_additional_security_groups: "{{ asg_additional_security_groups_env | default
 
 
 ecs_cluster_name: "{{ cluster_name }}"
+
+
+efs_commands: "sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 $(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone){{ efs_instance.efs.mount_point }} data"
```

Comment:
I think this should go in the readme you will create in custom-infrastructure. This is not a core module so the examples shouldn't include extra modules